src/main/java/net/dancier/resources/login/LoginResource.java

```diff
@@ -139,7 +139,7 @@ public FacebookProfile() {}
     @Path("callback")
     public Response callback(@Context HttpServletRequest request, @Context ContainerRequestContext requestContext) {
         logger.debug("Received callback");
-        if (loginConfiguration.devEnv) {
+        if (loginConfiguration.devEnv && !getParam(request.getParameterMap(), OIDC_PARAM_CODE).isPresent()) {
```

Comment:
Without this I had the situation, that wen devEnv is enabled, any real oauth-redirect stopped working completly

Reply:
I already approved 11 days ago ;-) But I can once again after the new commit of course.